src/rpc/blockchain.cpp

```diff
@@ -466,7 +466,10 @@ UniValue mempoolToJSON(bool fVerbose)
             const uint256& hash = e.GetTx().GetHash();
             UniValue info(UniValue::VOBJ);
             entryToJSON(info, e);
-            o.pushKV(hash.ToString(), info);
+            UniValue entry(UniValue::VOBJ);
+            entry.pushKV(hash.ToString(), info);
+            // `pushKVs` is used to avoid duplicate key check in `pushKV`
```

Comment:
```suggestion
            // pushKVs is used to avoid duplicate key check in pushKV
            // It is safe to use because ::mempool.mapTx is a set without duplicates
```